profile/src/main/java/rdx/works/profile/data/model/extensions/EntityExtensions.kt

```diff
@@ -112,3 +112,7 @@ fun Profile.unhideAllEntities(): Profile {
 fun Entity.isNotHidden(): Boolean {
     return flags.contains(EntityFlag.DeletedByUser).not()
 }
+
+fun Entity.isHidden(): Boolean {
+    return flags.contains(EntityFlag.DeletedByUser)
+}
```

Comment:
Maybe we just need one method for that like `isHidden()` and we can just negate it whenever we need the not hidden entities.